<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushDock</class>
 <widget class="QWidget" name="BrushDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>370</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="widgets::BrushPreview" name="preview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QWidget" name="brushTypeContainer" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="widgets::GroupedToolButton" name="hardedgeMode">
          <property name="toolTip">
           <string>Pixel brush</string>
          </property>
          <property name="icon">
           <iconset theme="drawpile_pixelround">
            <normaloff>theme:drawpile_pixelround.svg</normaloff>theme:drawpile_pixelround.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="groupPosition">
           <enum>widgets::GroupedToolButton::GroupLeft</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="widgets::GroupedToolButton" name="squareMode">
          <property name="toolTip">
           <string>Square pixel brush</string>
          </property>
          <property name="icon">
           <iconset theme="drawpile_square">
            <normaloff>theme:drawpile_square.svg</normaloff>theme:drawpile_square.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="groupPosition">
           <enum>widgets::GroupedToolButton::GroupCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="widgets::GroupedToolButton" name="softedgeMode">
          <property name="toolTip">
           <string>Brush</string>
          </property>
          <property name="icon">
           <iconset theme="drawpile_round">
            <normaloff>theme:drawpile_round.svg</normaloff>theme:drawpile_round.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="groupPosition">
           <enum>widgets::GroupedToolButton::GroupCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="widgets::GroupedToolButton" name="mypaintMode">
          <property name="toolTip">
           <string>MyPaint Brush</string>
          </property>
          <property name="icon">
           <iconset theme="draw-brush">
            <normaloff>theme:draw-brush.svg</normaloff>theme:draw-brush.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="groupPosition">
           <enum>widgets::GroupedToolButton::GroupRight</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="blendmode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::GroupedToolButton" name="modeColorpick">
       <property name="toolTip">
        <string>Pick initial color from layer</string>
       </property>
       <property name="icon">
        <iconset theme="color-picker">
         <normaloff>theme:color-picker.svg</normaloff>theme:color-picker.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::GroupedToolButton" name="modeLockAlpha">
       <property name="toolTip">
        <string>Lock alpha (recolor mode)</string>
       </property>
       <property name="icon">
        <iconset theme="color-picker">
         <normaloff>theme:object-locked.svg</normaloff>theme:object-locked.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::GroupedToolButton" name="modeEraser">
       <property name="toolTip">
        <string>Eraser mode</string>
       </property>
       <property name="icon">
        <iconset theme="draw-eraser">
         <normaloff>theme:draw-eraser.svg</normaloff>theme:draw-eraser.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::GroupedToolButton" name="modeIncremental">
       <property name="toolTip">
        <string>Direct drawing mode</string>
       </property>
       <property name="icon">
        <iconset theme="drawpile_incremental_mode">
         <normaloff>theme:drawpile_incremental_mode.svg</normaloff>theme:drawpile_incremental_mode.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,0">
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="4" column="3">
      <widget class="widgets::GroupedToolButton" name="pressureSmudging">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="draw-brush">
         <normaloff>theme:draw-brush.svg</normaloff>theme:draw-brush.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="widgets::GroupedToolButton" name="pressureHardness">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="draw-brush">
         <normaloff>theme:draw-brush.svg</normaloff>theme:draw-brush.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="widgets::GroupedToolButton" name="pressureOpacity">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="draw-brush">
         <normaloff>theme:draw-brush.svg</normaloff>theme:draw-brush.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QComboBox" name="inputPreset">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Input: </string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="widgets::GroupedToolButton" name="configureInput">
       <property name="toolTip">
        <string>Configure input presets</string>
       </property>
       <property name="icon">
        <iconset theme="configure">
         <normaloff>theme:configure.svg</normaloff>theme:configure.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="widgets::GroupedToolButton" name="pressureSize">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="draw-brush">
         <normaloff>theme:draw-brush.svg</normaloff>theme:draw-brush.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="brushsizeBox">
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="prefix">
        <string>Size: </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="radiusLogarithmicBox">
       <property name="prefix">
        <string>Radius: </string>
       </property>
       <property name="maximum">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="opacityBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="prefix">
        <string>Opacity: </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="hardnessBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="prefix">
        <string>Hardness: </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="smudgingBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="prefix">
        <string>Smudging: </string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="colorpickupBox">
       <property name="prefix">
        <string>Color Pickup: 1/</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="brushspacingBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="prefix">
        <string>Spacing: </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="gainBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string>Gain: </string>
       </property>
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="KisSliderSpinBox" name="slowTrackingBox">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="prefix">
        <string>Stabilizer: </string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgets::BrushPreview</class>
   <extends>QFrame</extends>
   <header>desktop/widgets/brushpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>widgets::GroupedToolButton</class>
   <extends>QToolButton</extends>
   <header>desktop/widgets/groupedtoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/kis_slider_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hardedgeMode</tabstop>
  <tabstop>squareMode</tabstop>
  <tabstop>softedgeMode</tabstop>
  <tabstop>mypaintMode</tabstop>
  <tabstop>blendmode</tabstop>
  <tabstop>modeColorpick</tabstop>
  <tabstop>modeLockAlpha</tabstop>
  <tabstop>modeEraser</tabstop>
  <tabstop>modeIncremental</tabstop>
  <tabstop>brushsizeBox</tabstop>
  <tabstop>pressureSize</tabstop>
  <tabstop>radiusLogarithmicBox</tabstop>
  <tabstop>opacityBox</tabstop>
  <tabstop>pressureOpacity</tabstop>
  <tabstop>hardnessBox</tabstop>
  <tabstop>pressureHardness</tabstop>
  <tabstop>smudgingBox</tabstop>
  <tabstop>pressureSmudging</tabstop>
  <tabstop>colorpickupBox</tabstop>
  <tabstop>brushspacingBox</tabstop>
  <tabstop>gainBox</tabstop>
  <tabstop>slowTrackingBox</tabstop>
  <tabstop>inputPreset</tabstop>
  <tabstop>configureInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
